src/plugins/integrations/MySQL2Integration.ts

```diff
@@ -40,68 +42,73 @@ class MySQL2Integration implements Integration {
   wrap(lib: any, config: any) {
     function wrapper(query: any) {
       return function queryWrapper(sql: any, values: any, cb: any) {
-        const tracer = ThundraTracer.getInstance();
-        const parentSpan = tracer.getActiveSpan();
-
-        const span = tracer._startSpan(this.config.database, {
-          childOf: parentSpan,
-          domainName: DomainNames.DB,
-          className: ClassNames.RDB,
-          disableActiveStart: true,
-        });
-
-        span.addTags({
-          [SpanTags.SPAN_TYPE]: SpanTypes.RDB,
-          [DBTags.DB_INSTANCE]: this.config.database,
-          [DBTags.DB_USER]: this.config.user,
-          [DBTags.DB_HOST]: this.config.host,
-          [DBTags.DB_PORT]: this.config.port,
-          [DBTags.DB_TYPE]: DBTypes.MYSQL,
-          [SpanTags.TOPOLOGY_VERTEX]: true,
-          [SpanTags.TRIGGER_DOMAIN_NAME]: LAMBDA_APPLICATION_DOMAIN_NAME,
-          [SpanTags.TRIGGER_CLASS_NAME]: LAMBDA_APPLICATION_CLASS_NAME,
-          [SpanTags.TRIGGER_OPERATION_NAMES]: [tracer.functionName],
-        });
-
-        const sequence = query.call(this, sql, values, cb);
-
-        const statement = sequence.sql;
-
-        if (statement) {
-          const statementType = statement.split(' ')[0].toUpperCase();
+        try {
+          const tracer = ThundraTracer.getInstance();
+          const parentSpan = tracer.getActiveSpan();
+
+          const span = tracer._startSpan(this.config.database, {
+            childOf: parentSpan,
+            domainName: DomainNames.DB,
+            className: ClassNames.RDB,
```

Comment:
I think, anymore we can set class name to the `DBTypes.MYSQL` here instead of `RDB` constant.

Reply:
Done. Classname is set `MYSQL` now.